<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgTotal</class>
 <widget class="QWidget" name="wdgTotal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Informe Total</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabDatos">
        <attribute name="title">
         <string>Datos</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Selecciona un año</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbYears"/>
             </item>
             <item>
              <widget class="QToolButton" name="cmd">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../images/xulpymoney.qrc">
                 <normaloff>:/xulpymoney/document-preview-archive.png</normaloff>:/xulpymoney/document-preview-archive.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="myQTableWidget" name="table">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectItems</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <row>
              <property name="text">
               <string>Ingresos</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Consolidado</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Dividendos</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Gastos</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>I+C+D-G</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Cuentas</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Inversiones</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Total</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Diferencia mensual</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string>% Variación principio año</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Enero</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Febrero</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Marzo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Abril</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Mayo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Junio</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Julio</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Agosto</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Septiembre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Octubre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Noviembre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Diciembre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Total</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabGraphic">
        <attribute name="title">
         <string>Gráfico</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="tabGraphTotal">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Selecciona un año</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbYearsG"/>
             </item>
             <item>
              <widget class="QToolButton" name="cmdG">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../images/xulpymoney.qrc">
                 <normaloff>:/xulpymoney/document-preview-archive.png</normaloff>:/xulpymoney/document-preview-archive.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
